source/includes/basics.md

```diff
@@ -82,13 +64,35 @@ Here's how to verify your API Key:
 
 Ping the API at `http://api.namara.io/v0/users/verify_token` to ensure that the API token is valid for the user.
 
->1)
+>**1)**
 `{ "status": true }`
 
->2)
+>**2)**
 `{ "status": false }`
 
 Response | Description
 -------- | -----------
 200: OK | The API Key is valid (*example 1*)
-422: Unprocessable Entity | API does not correspond to any user (*example 2*)
\ No newline at end of file
+422: Unprocessable Entity | API does not correspond to any user (*example 2*)
+
+### Rate Limiting
+
+Users are limited to 10,000 requests per month, as well as 100 data set downloads per month. When you exceed this limit, the API will return `status: 429`. If you find yourself meeting the limits, contact your account manager or <a href="mailto:info@thinkdataworks.com" target="_blank" rel="noreferrer noopener">email us</a> to find a solution.
+
+# API Keys
```

Comment:
Should this section on API Keys be higher in the docs? I mean everything in the docs relies on the user knowing / having access to their key. Thoughts?

Reply:
This was pulled out of the "intro" section specifically to give it more visibility in the sidebar. It was higher in that intro section, and Chris gave me feedback from hackathoners who said that info was hard to find.